packages/dso-toolkit/src/components/form/form-group-collection.scss

```diff
@@ -11,8 +11,13 @@
     margin-block-end: units.$u2;
 
     > legend {
-      border-block-end: 0;
-      margin-block-end: css-form-variables.$fieldset-separator-unit;
+      @include headings.h3();
```

Comment:
Ik denk dat dit een merge conflict die verkeerd is resolved:

![image](https://github.com/user-attachments/assets/85f88cb5-fb63-48de-a086-9640a237e599)

Reply:
gefixt nu denk ik